docs/reference/search/suggesters/completion-suggest.asciidoc

```diff
@@ -238,3 +209,40 @@ The following parameters are supported:
 NOTE: If you want to stick with the default values, but
```

Comment:
On `unicode_aware`, just above this say:

    If `true`, all measurements (like fuzzy edit distance, transpositions, and lengths) are measured in Unicode 
    code points instead of in bytes.  This is slightly slower than raw bytes, so it is set to `false` by default.